### PR TITLE
18346

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)

"Visible Section Not Found" message appears if Magento admin session times out. It must show message like you were logged out due to inactivity

### Fixed Issues (if relevant)

1. magento/magento2#18346: "Visible Section Not Found" message appears if Magento admin session times out.

### Manual testing scenarios (*)

1. Login to admin panel.
2. Wait for session timeout.
3. Reload the page.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/design/adminhtml/Magento/backend/web/css/source/actions/_actions-select.less [hunk 1]
```diff
@@ -109,7 +109,7 @@
     .action-menu-items {
         left: 0;
         position: absolute;
-        right: 0;
+        right: auto;
```
Comment (dmytro-ch) [inline app/design/adminhtml/Magento/backend/web/css/source/actions/_actions-select.less:112]: I guess, this change is not related to the current issue.

File: app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php [hunk 2]
```diff
@@ -31,22 +32,30 @@ abstract class AbstractConfig extends \Magento\Backend\App\AbstractAction
     protected $_configStructure;
 
     /**
-     * @deprecated 100.2.0
+     * @var AdminSessionsManager
+     */
+    protected $_sessionsManager;
```
Comment (dmytro-ch) [inline app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php:37]: All the new properties must be declared in the private scope of visibility.
Please also don't use underscores in property names, this is a legacy approach.

File: app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php [hunk 3]
```diff
@@ -31,22 +32,30 @@ abstract class AbstractConfig extends \Magento\Backend\App\AbstractAction
...
     /**
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\Config\Model\Config\Structure $configStructure
+     * @param AdminSessionsManager $sessionsManager
      * @param mixed $sectionChecker - deprecated
      */
     public function __construct(
         \Magento\Backend\App\Action\Context $context,
         \Magento\Config\Model\Config\Structure $configStructure,
+        AdminSessionsManager $sessionsManager,
```
Comment (dmytro-ch) [inline app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php:53]: The new constructor parameters should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter) (section *Adding a constructor parameter*) for more details.
Thus you won't need to update the classes which inherit this one.

File: app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php [hunk 4]
```diff
@@ -58,6 +67,10 @@ public function __construct(
      */
     public function dispatch(\Magento\Framework\App\RequestInterface $request)
     {
+        if (!$this->_sessionsManager->getCurrentSession()->isLoggedInStatus()) {
+            $this->messageManager->addSuccessMessage(__('You were logged out due to inactivity.'));
```
Comment (dmytro-ch) [inline app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php:71]: Please use the default message, which already has translations: "Your current session has been expired."

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hi @vishesh-webkul,
could you please fix the code review notices and automated tests failures?

I also have doubts about the current solution.
The actual problem is caused by the incorrect behavior starting from `\Magento\Security\Model\Plugin\AuthSession::aroundProlong` plugin method. I would recommend investigating and fixing the root reason of the bug instead of applying the workarounds.

Thank you!

**Comment by ghost** [comment]
Please close this pull request it has some error by mistake. I will create a new PR.
